src/azure/cli/_output.py

```diff
@@ -78,6 +54,20 @@ def format_text(obj):
     except TypeError:
         return ''
 
+def format_table(obj):
+    result = obj.result
+    try:
+        if obj.simple_output_query and not obj.is_query_active:
```

Comment:
>if obj.simple_output_query and not obj.is_query_active: [](start = 8, length = 55)

If you are going to ignore the callable if query is specified, I would recommend a warning log at a minimum indicating that's what's happening.